### ISSUE TITLE
`jx install` enhancement

### ISSUE THREAD

**Issue by ktrzeciaknubisa** [jxcore/jxcore#290]
There are some npm modules out there, which are using [scripts](https://docs.npmjs.com/misc/scripts) property of package.json file. There might be defined a command in there to be launched with `node` executable. Especially `install` and `preinstall` might decide whether package installation will fail or not.  For example:

```js
 "scripts": {
    "install": "node tools/install.js"
  }
```
The above fragment is taken from [edge](https://github.com/tjanczuk/edge) module.

Since `jx install` does not require node to be installed, the package installation fails, when `node`is not available within a `%PATH%`.

One solution would be to modify `package.json` on the fly and instead of `node` insert `/path/to/jx` at least for `install` and `preinstall` values.

However this is just a part of a problem, since post installation commands can be also defined in .gyp files, which is in fact the only supported way of doing so (as stated under the above link to scripts).

Thus I'm marking this issue just as an enhancement and waiting for other opinions.

**Comment by obastemur** [comment]
'jx install' replaces most of the node references on target modules to jx. This is also something we can take care of on our patched version of npm.

**Comment by ktrzeciaknubisa** [comment]
The above commit replaces `node` occurrences in package.json `scripts` section. Apart from [edge](https://github.com/tjanczuk/edge) module mentioned in first comment (which is now installable on Windows), I've noticed that `jx install phantomjs@1.9.17` didn't work even on OSX. Now it works.

However the commit solves the problem only partially, since as I wrote before:

> this is just a part of a problem, since post installation commands can be also defined in .gyp files, which is in fact the only supported way of doing so 

So this needs to be checked further.
